certora/README.md

```diff
@@ -0,0 +1,51 @@
+# Certora 
+
+This folder provides a fully functional skeleton for integrating with Certora.
+There are two configurations:
+
+
+
+## Run From the Command Line
+
+To run the checks from the command line, first install the Python SDK:
+`pip install certora-cli`
+
+Set your Certora Key:  
+`export CERTORAKEY=<personal_access_key>` 
```

Comment:
isnt this only for a given session of a terminal?
Shouldnt we give instructions to add to .zshenv or something?

Reply:
done